build.gradle

```diff
@@ -92,7 +90,7 @@ subprojects {
 	}
 
 	dependencyManagement {
-		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}" //mavenBom "org.springframework.cloud:spring-cloud-starter-parent:${springCloudStarterParentTrain}"
+		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springCloudStarterParentTrain}"
```

Comment:
I can't find a org.springframework.boot:spring-boot-starter-parent:Brixton.M3 in any repos.

note the `boot` rather than `cloud`